lib/split/dashboard.rb

```diff
@@ -17,58 +18,78 @@ class Dashboard < Sinatra::Base
     helpers Split::DashboardHelpers
     helpers Split::DashboardPaginationHelpers
 
-    get '/' do
+    get "/" do
       # Display experiments without a winner at the top of the dashboard
       @experiments = Split::ExperimentCatalog.all_active_first
+      @unintialized_experiments = Split.configuration.experiments.keys - @experiments.map(&:name)
 
       @metrics = Split::Metric.all
 
       # Display Rails Environment mode (or Rack version if not using Rails)
-      if Object.const_defined?('Rails')
+      if Object.const_defined?("Rails")
         @current_env = Rails.env.titlecase
       else
         @current_env = "Rack: #{Rack.version}"
       end
       erb :index
     end
 
-    post '/force_alternative' do
+    post "/initialize_experiment" do
+      Split::ExperimentCatalog.find_or_create(params[:experiment]) unless params[:experiment].nil? || params[:experiment].empty?
+      redirect url("/")
+    end
```

Comment:
Not sure why this is appearing as a diff as this was already added by me in our forked repo ~ maybe since it was added separately to the public repo and being merged back in 🤷‍♂️